### PR TITLE
chore: add Tip Calculator #1277

### LINKED ISSUE #1277: [Add a Play]: tip calculator for bills
### What's the Play about?

A simple react app to calculate tip evenly where user can divide the total bill amount into sharing number of people and the tip, and distribute how much the bill is for per person

### What ReactJS concept will be used to create this play?

ReactJs concepts: useState, useEffect

### Record

- [X] I have checked the existing [issues](https://github.com/reactplay/react-play/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/reactplay/react-play/blob/main/CONTRIBUTING.md)
- [X] I want to work on this issue

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ayushsgithub**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Adds a Tip Calculator in the ReactPlay

Fixes #1277 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Tested Manually

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

Screenshot 2023-10-06 175720

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
please make sure to rename the file extensions to `jsx`.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

File: src/plays/tip-calculator/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* enter stlyes here */
+:root {
```
Comment (atapas) [inline src/plays/tip-calculator/styles.css:2]: @ayushsgithub Let's not override the :root selector. Could you please add a style in your play level instead? Rest of the changes look good?

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/plays/tip-calculator/styles.css (hunk 1 above)
Comment (ayushsgithub) [reply src/plays/tip-calculator/styles.css]: Hey @atapas sir, removed the root selector as per the required changes

**Review by atapas — CHANGES_REQUESTED** [review]
@ayushsgithub very good work.

**Comment by ayushsgithub** [comment]
> @ayushsgithub very good work.

@atapas Thank You so much sir, I'm fan of your work and content.

**Comment by ayushsgithub** [comment]
@priyankarpal there's a request, can you please add hacktoberfest-accepted label to it?

**Pull request merged**